src/mscorlib/shared/System/Byte.cs

```diff
@@ -111,8 +111,20 @@ public static byte Parse(String s, NumberStyles style, IFormatProvider provider)
             return Parse(s, style, NumberFormatInfo.GetInstance(provider));
         }
 
+        public static byte Parse(ReadOnlySpan<char> s, NumberStyles style, IFormatProvider provider)
+        {
+            NumberFormatInfo.ValidateParseStyleInteger(style);
+            return Parse(s, style, NumberFormatInfo.GetInstance(provider));
+        }
+
         private static byte Parse(String s, NumberStyles style, NumberFormatInfo info)
         {
+            if (s == null) ThrowHelper.ThrowArgumentNullException(ExceptionArgument.s);
```

Comment:
@jkotas, I pushed a commit to make these methods consistent across the primitive types.